requirements.txt

```diff
@@ -10,6 +10,7 @@ python-dateutil==2.4.2
 requests==2.13.0
 newrelic==2.82.0.62
 gunicorn==19.7.1
+gevent==1.1.2
```

Comment:
Should we have 1.2.1 here instead?  We can adjust in a follow-up PR after reverting log level.